README.md

```diff
@@ -1,47 +1,185 @@
 # react-e [![CircleCI](https://circleci.com/gh/Originate/react-e.svg?style=svg)](https://circleci.com/gh/Originate/react-e)
 
-React helper to generate elements
+JSX replacement that doesn't need to be compiled.
 
-## Installing
-`npm install --save react-e`
+## Basic Usage
 
-## Example Usage
+```javascript
+e = require('react-e')
 
-```coffeescript
-e = require 'react-e'
-
-e '.container',
-  e 'button.primary', 'Click me'
-  e 'span.text', className: active: true, 'Some text'
+e(Modal,
+  e('.container',
```

Comment:
Do you feel strongly enough about this to dictate that none of your users can ever use destructuring?

What about providing an easy way to use this library without destructuring (as you already do), and recommend that in the readme. And also allowing to use destructuring if people want to? I have found that a little bit of prudence goes a long way with such issues. I use destructuring on prelude-ls, often with 3-5 elements, and it's not a problem at all.